pipeline/inputs/aggregator.go

```diff
@@ -223,21 +223,24 @@ type aggregatedReport metrics.MetricReport
 
 // accept possibly aggregates the given MetricReport into this aggregatedReport. Returns true
 // if the report was aggregated, or false if the labels or name don't match. Returns an error if the
-// given report could be aggregated (i.e., labels match), but its time range conflicts with the
-// existing aggregated range.
+// given report could be aggregated (i.e., labels match), but has earlier end time than the
+// aggregated end time.
```

Comment:
I added CI for this repo in #51. I will try to merge it asap so that this PR can benefit from it.

For your own iteration, please run `make setup deps test` prior to submission.